go.mod

```diff
@@ -389,7 +389,7 @@ replace (
 	github.com/datastax/go-cassandra-native-protocol => github.com/gravitational/go-cassandra-native-protocol v0.0.0-20221005103706-b9e66c056e90
 	github.com/go-mysql-org/go-mysql => github.com/gravitational/go-mysql v1.5.0-teleport.1
 	github.com/go-redis/redis/v9 => github.com/gravitational/redis/v9 v9.0.0-teleport.3
-	github.com/gogo/protobuf => github.com/gravitational/protobuf v1.3.2-0.20201123192827-2b9fcfaffcbf
+	github.com/gogo/protobuf => github.com/gravitational/protobuf v1.3.2-1
```

Comment:
I might even go with a more explicit prerelease tag like `v1.3.2-teleport.1` to make it clear this is our work.

Reply:
Good recommendation, I see this pattern being used in the replacements above.  I have updated the tag to `v1.3.2-teleport.1`